### DISCUSSION TITLE
return selected file to termnal and allow user to configure how to open files

### DISCUSSION THREAD

**Discussion by bokunodev** [antonmedv/llama#150] [Ideas]
feature request:
1. add ability to return selected file to terminal by pressing `Shift+Enter` or `Shift+Q`.
    this will be very useful for scripting
2. allow user to configure on how to open files. for example using `exo-open` instead of `$EDITOR`.

**Comment by antonmedv** [comment]
> 1. add ability to return selected file to terminal by pressing Shift+Enter or Shift+Q.
this will be very useful for scripting

Nice idea.

> 2. allow user to configure on how to open files. for example using exo-open instead of $EDITOR.

Walk supports `WALK_EDITOR`.
